app/models/service.py

```diff
@@ -46,7 +47,7 @@ def create(uuid: str, device: str, owner: str, name: str) -> 'Service':
             running=config["services"][name]["auto_start"],
             name=name,
             running_port=config["services"][name]["default_port"],
-            consumption=config["services"][name]["consumption"]
+            consumption=config["services"][name]["consumption"],
```

Comment:
`config["services"][name]["consumption"]` has been removed from the config file

Reply:
done